assignment1/cube_sum.rb

```diff
@@ -0,0 +1,9 @@
+def sum_of_cubes num_range, power
+    sum = 0
+    num_range.each { |arr|
+        sum += arr ** power
+    }
+    puts sum
+end
+
+sum_of_cubes (1..4), 2
```

Comment:
@Kiransai-webonise can you please add a single line at the end also create a single .rb file and paste all your assignments there.